src/file/file.rpc.ts

```diff
@@ -41,30 +48,45 @@ export class FileServer {
     // Register the health service
     this.grpcHealthCheck = new GrpcHealthCheck(healthCheckStatusMap);
     this.server.addService(HealthService, this.grpcHealthCheck);
+    this.server.addService(
```

Comment:
Write that in a seperate function

Reply:
How can you separate function's call from its params? Either make the params as `const` or `class` somehow. At the end it should be like `this.server.addService(HealthService, this.grpcHealthCheck);`.